### PR TITLE
[macOS] Fix redundancy in VoiceOver announcements for groups with `IconButton`.

### PR DESCRIPTION
**Pull request by a-wallen**
## This patch is different than https://github.com/flutter/engine/pull/36599 as it returns an empty string.

[VoiceOver repeats same text or label property text of Semantics Widget on MacOS 12.5.1](https://github.com/flutter/flutter/issues/111094) when `AXTitleInternal` is called twice from `accessibilityTitle` and `accessibilityLabel`. 

**Demo**

https://user-images.githubusercontent.com/44445638/193924691-1e74891b-4999-4fda-bde1-a69438fb2de9.mov

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [Flutter Style Guide] and the [C++, Objective-C, Java style guides].
- [x] I listed at least one issue that this PR fixes in the description above.
- [ ] I added new tests to check the change I am making or feature I am adding, or Hixie said the PR is test-exempt. See [testing the engine] for instructions on
writing and running engine tests.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [x] I signed the [CLA].
- [ ] All existing and new tests are passing.

### CODE REVIEW

File: third_party/accessibility/ax/platform/ax_platform_node_mac.mm [hunk 1]
```diff
@@ -767,7 +767,7 @@ - (NSString*)accessibilityLabel {
   // and accessibilityTitle is "the title of the accessibility element"; at
   // least in Chromium, the title usually is a short description of the element,
   // so it also functions as a label.
-  return [self AXTitleInternal];
```
Comment (chunhtai) [inline third_party/accessibility/ax/platform/ax_platform_node_mac.mm:770]: Is there a reason to mute the label over title?

Comment (a-wallen) [reply third_party/accessibility/ax/platform/ax_platform_node_mac.mm:770]: According to the comment, I shouldn't do it :)

Comment (a-wallen) [reply third_party/accessibility/ax/platform/ax_platform_node_mac.mm:770]: > // least in Chromium, the title usually is a short description of the element,
  // so it also functions as a label.

**Review by chunhtai — APPROVED** [review]
LGTM

File: third_party/accessibility/ax/platform/ax_platform_node_mac.mm (hunk 1 above)
Comment (chunhtai) [reply third_party/accessibility/ax/platform/ax_platform_node_mac.mm:770]: probably add a doc mentions why this is an empty string

**Comment by a-wallen** [comment]
Closing as I think that finishing the implementation of https://github.com/flutter/engine/pull/36599/files yield a longer lasting solution than this patch.

**Pull request closed without merging**